### DISCUSSION TITLE
Installation Setup

### DISCUSSION THREAD

**Discussion by rodrigoricky** [retypeapp/retype#178] [General]
I just wanna ask on how to setup it like which folder should I put the markdown files and will this run on github pages?

**Comment by geoffreymcgill** [comment]
Hi @rodrigoricky. The best place to start is the [Getting Started](https://retype.com/guides/getting-started/) guide.

Retype is still a command line only tool, so a little more developer focused, but the steps are super simple:

1. [Install](https://retype.com/guides/getting-started/) Retype using the command line
2. Move the command line prompt to the folder with your Markdown `.md` files
3. Run the command `retype watch`

Your Markdown `.md` files can be placed in any folder. Whatever location is most convenient for you.

Websites built by Retype are easily setup to run on GitHub Pages. See our [guide](https://retype.com/guides/github-actions/).

If you have any issues getting Retype working, please feel free to post them here and we will do our best to assist.

Hope this helps.

**Reply by rodrigoricky** [reply]
Hey, how do I add a custom web title in the website without it being synced to the page title?

**Comment by geoffreymcgill** [comment]
Hi @rodrigoricky. You can set a custom [`title`](https://retype.com/configuration/page/#title) config in the page metadata.

```md
---
title: Your custom title here
---
# Welcome

This is a sample paragraph.
```

Hope this helps
